### PR TITLE
ch-docker2tar: add progressbar using pv

### LINKED ISSUE #148: docker2tar: use pv for status reporting
docker2tar can take a while, using `pv` (http://www.ivarch.com/programs/pv.shtml) might help users to stay patient!

### PR DESCRIPTION
**Pull request by junghans**
Fix #148

### CODE REVIEW

**Comment by junghans** [comment]
We could install `pv` in Travis, too!

File: bin/base.sh [hunk 1]
```diff
@@ -16,3 +16,13 @@ if ( command -v pigz >/dev/null 2>&1 ); then
 else
     export GZIP_CMD=gzip
 fi
+
+if ( command -v pv >/dev/null 2>&1 ); then
```
Comment (reidpr) [inline bin/base.sh]: What if we also add to this condition, only do `pv` if we are on a terminal?

File: bin/base.sh [hunk 2]
```diff
@@ -16,3 +16,13 @@ if ( command -v pigz >/dev/null 2>&1 ); then
...
+    PV() {
+        pv "$@"
+    }
+else
+   PV() {
+       cat
```
Comment (reidpr) [inline bin/base.sh]: Can you add a comment her saying that the arguments are being ignored and why?

File: bin/base.sh (hunk 1 above)
Comment (junghans) [reply bin/base.sh]: Done

File: bin/base.sh (hunk 2 above)
Comment (junghans) [reply bin/base.sh]: Done

**Comment by junghans** [comment]
> Thanks Christof. This helps!

`s/f/ph/`

> In addition to the in-line comments, do you have time to also:
> 
> Review the other scripts and see if there are more opportunities for this trick (e.g. ch-tar2dir)?

I did `ch-tar2dir` as well.

> Add yourself to the contributors list in the readme?

Done

**Pull request merged**